src/common/input-validation/PackageIdValidationInput.jsx

```diff
@@ -1,24 +1,44 @@
 import React from "react";
-import {FormControl, FormHelperText, Input, InputLabel, withStyles} from "@material-ui/core";
+import { styled } from '@mui/material/styles';
+import {
+    FormControl,
+    FormHelperText,
+    Input,
+    InputLabel,
+    Grid,
+    Typography
+} from "@mui/material";
 import PropTypes from "prop-types";
-import Grid from "@material-ui/core/Grid/Grid";
-import Typography from "@material-ui/core/Typography/Typography";
 
-const styles = theme => ({
-    realm: {
+const PREFIX = 'PackageIdValidationInput';
+
+const classes = {
+    realm: `${PREFIX}-realm`,
+    label: `${PREFIX}-label`,
+    input: `${PREFIX}-input`
+};
+
+const StyledFormControl = styled(FormControl)((
+    {
+        theme
```

Comment:
seeing this over 3 lines a lot. is this automatically formatted? takes up a lot of space 😛